Add new cruise mode to modes ordering

MrD-RC opened the pull request:
I thought I left Cruise in there. It wasn't, so it's back now we have a Cruise mode again.

Jetrell commented:
@MrD-RC Does this PR also prevent NavCruise and Failsafe modes from highlighting blue when not active?

![modes](https://user-images.githubusercontent.com/47995726/118070138-afe90280-b3e8-11eb-96ac-1801dce791cc.JPG)

MrD-RC commented:
@Jetrell at first glance, that doesn't appear to be caused by the ordering. I removed the ordering code, and both remain highlighted.
![Modes](https://user-images.githubusercontent.com/17590174/118093183-004e6900-b3c5-11eb-8008-f44c3bc04926.png)

I added the IDs as debug, and the ID matches the original ID, so no ordering has taken place.
![modes 2](https://user-images.githubusercontent.com/17590174/118093558-85398280-b3c5-11eb-9c6c-bcd7e648118f.jpg)
...and the ordered version for comparison
![modes 3](https://user-images.githubusercontent.com/17590174/118093834-e5c8bf80-b3c5-11eb-815f-69b3c0b79941.jpg)
The highlighting is based on the original ID.

I also removed every range except for those 2, and they stay highlighted, no matter what channel I select, or where I move the ranges. I think this is something in the firmware causing it.

Jetrell commented:
@MrD-RC That is odd. Thank you for looking at it. Maybe it can be fixed at some later time.  
For now its good to have the two Cruise modes together. :+1:
